src/profile-logic/gecko-profile-versioning.js

```diff
@@ -328,7 +328,7 @@ const _upgraders = {
 
     function convertToVersionNineRecursive(p) {
       for (const thread of p.threads) {
-        //const stringTable = new UniqueStringArray(thread.stringTable);
+        //const stringTable = new StringTable(thread.stringTable);
```

Comment:
huh, I don't remember why we kept these as comments, but it's an old upgrader that we don't usually touch, so probably it's fine to keep.